packages/vscode-extension/src/webview/hooks/useBuildErrorAlert.tsx

```diff
@@ -85,15 +85,15 @@ function BundleErrorActions() {
       <IconButton
         type="secondary"
         onClick={() => {
-          project.focusDebugConsole();
+          project.focusExtensionLogsOutput();
         }}
-        tooltip={{ label: "Open debug console", side: "bottom" }}>
-        <span className="codicon codicon-debug-console" />
+        tooltip={{ label: "Open extension logs", side: "bottom" }}>
+        <span className="codicon codicon-output" />
       </IconButton>
       <IconButton
         type="secondary"
         onClick={() => {
-          project.reload("reloadJs");
+          project.restart(false);
```

Comment:
This is confusing. Why we use `false` in place of restart type?

Reply:
Restart takes a boolean meaning whether it should force clean build or not

Reply:
My bad, I didn't notice different method name.